gocd/gocd.go

```diff
@@ -79,6 +79,29 @@ type Pipeline struct {
 	Timer        interface{} `json:"timer"`
 }
 
+// Server a representstion of a GoCD server
+type Server struct {
+	GOCDHost     string
```

Comment:
These can omit the GOCD prefix, since the context can be assumed by the package name.